JenkinsfileRT

```diff
@@ -39,14 +39,12 @@ bc0.conda_packages = [
     "python=${python_version}",
 ]
 bc0.build_cmds = [
-    "pip install -e .[test]",
+    "pip install -e .[test,ephem]",
     "pip install pytest-xdist",
 ]
 bc0.test_cmds = [
-    "pytest -r sx -v -n 30 --bigdata --slow \
-    --basetemp=${pytest_basetemp}  --junit-xml=results.xml \
-    --log-level=INFO \
-    jwst/tests_nightly/general"
+    "pytest -r sxf -n 15 --bigdata --slow \
```

Comment:
`30` -> `15`: was this in case of RT jobs overlapping, or just so the host is still semi-usable for other things during runs?

Reply:
This was just in anticipation of our eventual use of a dev build/test running in parallel.  And it turns out 15 or 30 has no effect on total run time as it is a few tests that take forever that drive that.

Reply:
But I see we are doing this in other PRs, so I will nix this.